src/python_testing/TC_ZONEMGMT_2_4.py

```diff
@@ -108,6 +108,14 @@ def steps_TC_ZONEMGMT_2_4(self) -> list[TestStep]:
                      "Verify that the DUT responds with a NotFound status code"),
         ]
 
+    async def _trigger_motion_event(self, zone_id):
+        # CI: Use app pipe to trigger zone event.
+        # Manual: User should trigger a motion event from the defined zone.
+        if self.is_pics_sdk_ci_only:
+            self.write_to_app_pipe({"Name": "ZoneTriggered", "ZoneId": zone_id})
+        else:
+            self.wait_for_user_input(prompt_msg=f"Press enter and generate motion triggered event from zone {zone_id}")
```

Comment:
How the motion event is generated in manual test? I guess this is via vendor specific way, why press enter first?